### PR TITLE
INT-1070 Implement the availability settings menu item and page

### PR DESCRIPTION
**Pull request by nurutdinov**

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -3598,9 +3737,26 @@ components:
...
-        - id
-        - email
+        voicemail:
+          $ref: "#/components/schemas/AudioFile"
+        defaultPhoneNumber:
+          $ref: "#/components/schemas/PhoneNumber"
+        defaultCallForwarding:
+          $ref: "#/components/schemas/CallForwarding"
+    UserParams:
+      type: object
```
Comment (olyavertwist) [inline talk/talk-api.yaml:3747]: description: Facilitates filtering.

**Review by olyavertwist — CHANGES_REQUESTED** [review]
Added descriptions.

File: talk/talk-api.yaml [hunk 2]
```diff
@@ -3577,12 +3666,62 @@ components:
...
       type: object
       description: Information about Inperium Talk user.
+      required:
+        - id
+        - email
+        - status
       properties:
         id:
           $ref: "#/components/schemas/Id"
+        status:
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:3723]: are you sure that you want to return all user information from hub just to retrieve user configuration about statuses and call forwarding?

**Review by BullDog57Rus — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml (hunk 2 above)
Comment (nurutdinov) [reply talk/talk-api.yaml:3743]: I've added list of fields to request, see diff

**Review by olyavertwist — APPROVED** [review]
Left a small suggestion and also wrote you personally about the trace HTTP verb

**Review by BullDog57Rus — APPROVED** [review]

**Pull request merged**